### ISSUE TITLE
Don't work 005_CreatePayout

### ISSUE THREAD

**Issue by lvarash** [btcpayserver/btcpayserver-php-client#21]
Hello! Can someone tell me if a BTCPay server can be used for payouts?

**Comment by NicolasDorier** [comment]
No it can't, can you tell me in a nutshell what is the payout feature? Maybe we can replicate?

**Comment by lvarash** [comment]
Something like https://bitpay.com/api/?php#rest-api-resources-payouts

It would be great if we could upload private keys to send bitcoins via API

**Comment by NicolasDorier** [comment]
This would requires btcpayserver to know about the private keys though :/

**Comment by lvarash** [comment]
Why this is bad?
My knowledge of bitcoin is small. If there are other options for sending bitcoins via API, then I would really like to know.

If I want to receive and send bitcoins through a BTPPayServer, then what is the best way to organize this?

And can I access the bitcoind process in docker?

**Comment by woutersamaey** [comment]
I have created issues before where you would use the API to create transfers, but you’d need to sign them afterwards with a hardware wallet. So this would be a queue that needs approval. Just an idea though, no development has been done yet.

**Comment by lvarash** [comment]
I'm interesting to use a not hardware wallets

**Comment by NicolasDorier** [comment]
@woutersamaey yeah this is interesting idea, I could replicate both, Bitpay and blockchain info with this model. I will think about it, I think people would love it.

**Comment by NicolasDorier** [comment]
@Ivarash it is bad for btcpayserver to have private keys, because I want to sleep well I night knowing for sure that a bug or a security flaw can't drain the money of a user.

**Comment by lvarash** [comment]
@NicolasDorier ok, I'm understand you, no other ways to send bitcoins via API without hd wallet?

**Comment by NicolasDorier** [comment]
At the present no.
